Tests/designspaceLib/data/convert5to4_output/SourceSerif4Variable-Italic.designspace

```diff
@@ -196,7 +196,7 @@
       <kerning/>
       <info/>
     </instance>
-    <instance name="Source Serif 4 Bold Italic" familyname="Source Serif 4" stylename="Bold Italic" filename="Source Serif 4-Bold Italic.ttf" postscriptfontname="SourceSerif4Italic-Bold" stylemapfamilyname="Source Serif 4 Bold Italic" stylemapstylename="regular">
+    <instance name="Source Serif 4 Bold Italic" familyname="Source Serif 4" stylename="Bold Italic" filename="Source Serif 4-Bold Italic.ttf" postscriptfontname="SourceSerif4Italic-Bold" stylemapfamilyname="Source Serif 4" stylemapstylename="regular">
```

Comment:
This change sounds wrong, either keep the stylemapfamilyname, or change the stylemapstylename to bold italic?

Reply:
The DS has no style linking though? How is getStatName supposed to do the right thing?

Reply:
No style linking = stylemapfamilyname is set to the familyname + stylename, and stylemapstylename is set to regular

Reply:
Otherwise, applications that don't understand the 16/17 named ids are going to see a bunch of "Source Serif 4" fonts and won't be able to distinguish them, that's why we put the stylename into the stylemapfamilyname.

The other option, if the DS had style linking, would be to set the stylemapstylename correctly, in this case bold italic